src/elements/slider/slider.ts

```diff
@@ -207,16 +246,14 @@ export class SbbSliderElement extends SbbDisabledMixin(LitElement) {
           >
             <input
               tabindex="-1"
-              name=${this.name || nothing}
-              min=${this.min || nothing}
-              max=${this.max || nothing}
-              ?disabled=${this.disabled || this.readonly || nothing}
-              .valueAsNumber=${this.valueAsNumber || nothing}
+              min=${this.min}
+              max=${this.max}
+              ?disabled=${this.disabled || this.formDisabled || this.readonly || nothing}
```

Comment:
for boolean values, nothing is not needed (`?` does it already)
```suggestion
              ?disabled=${this.disabled || this.formDisabled || this.readonly}
```